testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-spark-iceberg/iceberg.properties

```diff
@@ -1,2 +1,4 @@
 connector.name=iceberg
 hive.metastore.uri=thrift://hadoop-master:9083
+# TODO: Remove this config to test default read behavior once Spark writer version is fixed. See https://github.com/trinodb/trino/issues/6369 for details
+iceberg.use-file-size-from-metadata=false
```

Comment:
Can this be removed now?
if Spark Iceberg still writes incorrect file sizes, we should reconsider whether `iceberg.use-file-size-from-metadata` shouldn't be `false` by default?